### PR TITLE
Add MFA app to Java Example Apps

### PR DESCRIPTION
**Pull request by awolfden**
Add MFA app to Java Example Apps

### CODE REVIEW

File: java-mfa-example/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    
+## Set up SSO with WorkOS
+7. Create an [Organization](https://dashboard.workos.com/organizations) and an [SSO Connection](https://workos.com/docs/sso/guide/introduction) in the Organization in your WorkOS Dashboard.
+8. Copy the Connection ID from the new connection that you just set up and add it to the same .env file that you created in step 6. The .env file should now look like this: 
+```bash
+WORKOS_API_KEY=your_api_key_here
+WORKOS_CLIENT_ID=your_project_id_here
+WORKOS_CONNECTION_ID=your_connection_id_here
+```
+
```
Comment (pliao28) [inline java-mfa-example/README.md:60]: These steps related to setting up SSO can be removed
```suggestion
## Set up SSO with WorkOS
7. Create an [Organization](https://dashboard.workos.com/organizations) and an [SSO Connection](https://workos.com/docs/sso/guide/introduction) in the Organization in your WorkOS Dashboard.
8. Copy the Connection ID from the new connection that you just set up and add it to the same .env file that you created in step 6. The .env file should now look like this: 
```bash
WORKOS_API_KEY=your_api_key_here
WORKOS_CLIENT_ID=your_project_id_here
WORKOS_CONNECTION_ID=your_connection_id_here
```

```

Comment (awolfden) [reply java-mfa-example/README.md:60]: Oh I forgot about the README! I'll update that today.

File: java-mfa-example/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+## Test the integration
+9. Start the server by running this command in the CLI while you’re in the java-example-applications/ directory:
+```bash
+$ ./gradlew :java-mfa-example:run
+```
+
+10. Navigate to http://localhost:7001. Click the “Login” button. You’ll be prompted to sign in using the identity provider (IdP) you configured during the SSO setup.
+
+11. Once you’re signed in, the webpage will display the user information for the logged-in user.
```
Comment (pliao28) [inline java-mfa-example/README.md:73]: No Login page for this example, can be tailored to mfa app or get rid of these steps

File: java-mfa-example/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Nice work! You just set up SSO for your Java app!
+
+## Need help?
+If you get stuck while following the steps below and aren't able to resolve the issue by reading our [API reference](https://workos.com/docs/reference) or [SSO Setup Guide](https://workos.com/docs/sso/guide), please reach out to us at support@workos.com so we can help!
```
Comment (pliao28) [inline java-mfa-example/README.md:78]: Refer to MFA docs instead of SSO docs

**Review by pliao28 — APPROVED** [review]
LGTM! Just the Readme can be edited to adjust for MFA instead of SSO otherwise it works for me!

**Pull request merged**